desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -104,6 +104,9 @@
   ## limit_request_fields=100
   ## limit_request_line=4094
 
+  # Flag to disable webpage caching
```

Comment:
Maybe also explicitly mention the performance issue of enabling this?

Reply:
mentioned about performance hit in the comment.